features/step_definitions/calculator_steps.rb

```diff
@@ -52,12 +71,33 @@
   expect(@page.calculator.pot_remaining).to have_content('£37,000')
 end
 
+Then(/^I should see an income that will last until my life expectancy$/) do
+  expect(@page.calculator.desired_income_lasts_until).to have_content('84')
+end
+
+Then(/^it shows me what age my desired income will last until$/) do
+  expect(@page.calculator.desired_income_lasts_until).to have_content('70')
+end
+
+And(/^it shows me a suggested monthly income that will last until an average life expectancy for someone of my age$/) do
+  content = 'An average life expectancy for someone born in the same year as you is 84 years. '\
+            'You could take a monthly income of £325 which would last until then.'
+
+  expect(@page.calculator.income_for_life_expectancy).to have_content(content)
+end
+
 Then(/^it explains the values are estimates based on growth at 3% per year$/) do
   content = 'This is an estimate based on your whole pot growing at a rate of about 3% per year — this may vary.'
```

Comment:
Same here with regards matching the whole content. The important bit is the growth interest rate.